wpilibc/src/generate/main/native/cpp/hid.cpp.jinja

```diff
@@ -90,3 +92,16 @@ bool {{ ConsoleName }}Controller::GetTouchpadReleased() {
   return GetRawButtonReleased(Button::kTouchpad);
 }
 {% endif %}
+
```

Comment:
Put a `-` on the right side of the `endif` to remove the extra line.

Reply:
You did two `-`, you just need one on the right side!

Reply:
whoops

Reply:
Actually I was wrong. We want to preserve the whitespace _after_ the `endif`, since that's fine. But since the `endif` itself is on a new line, that new line will stay. That means we want a `-` on the _left_ side of the `endif`, not the right side.